### PR TITLE
Add GuixSD to available distro packages in README.md

### PR DESCRIPTION
**Pull request by va511e**

### CODE REVIEW

**Comment by moneromooo-monero** [comment]
Please squash.

**Comment by va511e** [comment]
Squash done. Thanks for the notice.

File: README.md [hunk 1]
```diff
@@ -133,6 +133,14 @@ Installing a snap is very quick. Snaps are secure. They are isolated with all of
 
     xbps-install -S monero
 
+* GuixSD
+
+        # command line client and daemon
+        guix package -i monero
+
+        # gui client
+        guix package -i monero-core
```
Comment (moneromooo-monero) [inline README.md]: That bit should probably go in monero-core.

**Comment by va511e** [comment]
You are right, so I've done the changes and added a pr in monero-core too.

**Pull request merged**